django-stubs/urls/conf.pyi

```diff
@@ -33,6 +40,13 @@ def re_path(
     route: _StrOrPromise, view: Callable[..., HttpResponseBase], kwargs: dict[str, Any] = ..., name: str = ...
 ) -> URLPattern: ...
 @overload
+def re_path(
+    route: _StrOrPromise,
+    view: Callable[..., Coroutine[Any, Any, HttpResponseBase]],
```

Comment:
Should this be `Coroutine[None, None, HttpResponseBase]`? I don't think view coroutines should yield any values?

Reply:
Sounds fair enough